tests/unit/test_start_handlers.py

```diff
@@ -48,6 +48,40 @@ async def test_get_index_ni(self):
         self.assertEqual(contents.count('input--text'), 1)
         self.assertIn('type="submit"', contents)
 
+    @unittest_run_loop
+    async def test_post_index_en_with_retry_503(self):
+        with aioresponses(passthrough=[str(self.server._root)]) as mocked:
+            mocked.get(self.rhsvc_url, status=503)
```

Comment:
Appears to be line duplication here

Reply:
I needed to multiple mocks so that each retry returns a 503 the number of times it goes round the loop. However, I have recoded this to hopefully be more explicit.